README.rst

```diff
@@ -24,11 +24,12 @@ file of your project:
     {
         public function registerBundles()
         {
-            $bundles = array(
-                // ...
+            //...
+            if (in_array($this->getEnvironment(), ['dev', 'test'])) {
+                //...
+                $bundles[] = new Alex\DoctrineExtraBundle\AlexDoctrineExtraBundle();
```

Comment:
You forgot to fix my first commit, so here is a copy/paste:

> Because the vendor code could be not present, you should check if the bundle is here before using it.